orttraining/orttraining/python/orttraining_pybind_common.h

```diff
@@ -8,6 +8,9 @@
 #include "core/platform/env.h"
 #include <unordered_map>
 #include <cstdlib>
+#ifdef ENABLE_EAGER_MODE
+#include "orttraining/eager/ort_backends.h"
```

Comment:
do you really need to put it in common.h? i think you can put it in a seperate header and only include it in orttraining_python_module.cc with #ifdef guard for eager part. 
if you include it in common.h, you will have the risk to include the torch headers to many other cc files.

Reply:
Done, thanks